### ISSUE TITLE
Invalid adjoining-classes with pseudo element selector should detect as other error .

### ISSUE THREAD

**Issue by RyotaSugawara** [CSSLint/csslint#646]
I'm sorry, my English is not good.
I found invalid patterns which breaks css in Chrome 49.
Before Chrome 49, we can ignore this pattern ( browsers ignores bad pattern ), but now it breaks css.

For Example: 
```css
div:after, div::after.foo {
  content: "should be invalid pattern"
}
```

The code above is broken. Chrome 49 in this case ignores other selector.
Now CSSLint detect as adjoining-classes, but it should detect as other error.
